packages/datasource/src/api/client.js

```diff
@@ -61,4 +63,9 @@ client.interceptors.response.use(
   }
 );
 
+if (process?.env.NEXT_PUBLIC_RECAPTCHA_KEY) {
+  const getCaptcha = (config) => getCaptchaToken(process.env.NEXT_PUBLIC_RECAPTCHA_KEY, config.path);
+  client.interceptors.request.use(createCaptchaInterceptor('re_token_v3', getCaptcha));
```

Comment:
Does it mean the captcha API will be called for every request? Will there be any performance issue or throttling limit of calling the 3rd party captcha API?

Reply:
See packages/common/src/utils/axios.js:28:

```js
// This should call recaptcha api
client.post('/foo/bar', { re_token_v3: undefined })

// These wouldn't call recaptcha api
client.post('/foo/bar', { re_token_v3: 'xxx' })
client.post('/foo/bar', {})
```